common/ux-window.cpp

```diff
@@ -105,6 +105,10 @@ namespace rs2
 
         config_file::instance().set_nested_default(configurations::dds::enable_dds, false);
         config_file::instance().set_nested_default(configurations::dds::domain_id, 0);
+        
+        // Since we have seen on severan laptops models that using GLSL for processing cause a memory leak Decided to disable it by default
```

Comment:
Minor. If doing other changes in the file. Typo "severan". "Decided" does not need capital letter.